DigitalLearningSolutions.Data/Services/LoginService.cs

```diff
@@ -75,7 +75,8 @@ private LoginResult DetermineDestinationForSuccessfulLogin(UserEntity userEntity
                 userEntity.DelegateAccounts.SingleOrDefault(da => da.CentreId == singleCentreToLogUserInto.Value);
 
             var centreIsActive = adminAccountToLogInto?.CentreActive ?? delegateAccountToLogInto?.CentreActive ?? false;
-            var accountAtCentreIsActive = adminAccountToLogInto?.Active ?? delegateAccountToLogInto?.Active ?? false;
+            var accountAtCentreIsActive = (adminAccountToLogInto?.Active == null || adminAccountToLogInto.Active) &&
+                                          (delegateAccountToLogInto?.Active == null || delegateAccountToLogInto.Active);

```

Comment:
This diff leads to changes in behaviour in two circumstances:
* When adminAccountToLogInto.Active is true and delegateAccountToLogInto.Active is false, accountAtCentreIsActive is now false, was previously true
* When both are null, accountAtCentreIsActive is now true, was previously false.

What is the intended behaviour here? The second of those circumstances doesn't look right to me.

Reply:
- When adminAccountToLogInto.Active is true and delegateAccountToLogInto.Active is false, accountAtCentreIsActive is now false, was previously true. This is the intended change here, to fix the bug raised during testing where the produced true was incorrect (should not be able to log into either at the same centre when either is inactive)
- When both are null, accountAtCentreIsActive is now true, was previously false. This case isn't possible. They cannot both be null at this point, they would fall out at the `singleCentreToLogUserInto == null` return statement in that case.